### PR TITLE
[plugin.audio.radiorecord] 1.0.1

### PR DESCRIPTION
**Pull request by Nemiroff**
### Description
Update API
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.audio.radiorecord/addon.xml [hunk 1]
```diff
@@ -15,13 +15,14 @@
         <disclaimer lang="en_GB">This is not official addon</disclaimer>
         <disclaimer lang="ru_RU">Это не оффициальный аддон</disclaimer>
         <language>ru en</language>
-        <license>GNU General Public License v2.0 only</license>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline plugin.audio.radiorecord/addon.xml]: Please use a spdx compliant identifier. In this case: GPL-2.0-only

**Review by enen92 — CHANGES_REQUESTED** [review]
Just a minor

File: plugin.audio.radiorecord/addon.xml (hunk 1 above)
Comment (Nemiroff) [reply plugin.audio.radiorecord/addon.xml]: Done

**Review by enen92 — APPROVED** [review]
Would still be nice if you could use the identifier as the majority of the add-ons do now. Either way, since you don't touch the license line this is good to go

**Comment by Nemiroff** [comment]
> Would still be nice if you could use the identifier as the majority of the add-ons do now. Either way, since you don't touch the license line this is good to go

sorry but i don't understand what about you say

**Comment by enen92** [comment]
In your last commit you left the license string as it was originally. My request when I first reviewed this was for you to use a spdx compliant identifier (GPL-2.0-only) instead of your license string.

**Comment by Nemiroff** [comment]
> In your last commit you left the license string as it was originally. My request when I first reviewed this was for you to use a spdx compliant identifier (GPL-2.0-only) instead of your license string.

use `<license>GPL-2.0-only</license>` ?

**Comment by enen92** [comment]
Yeap. Keep it for the next update 🙂

**Pull request merged**